commit f3df6c78f885f9cd8ef8798c2654d790c39e70cd (refs/changes/00/4700/1)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-08-16T14:12:07-04:00 (1 year, 2 months ago)
    
    joyent/node-manta#303 mfind should probably require at least one path argument

diff --git a/bin/mfind b/bin/mfind
index 820582b..6d27942 100755
--- a/bin/mfind
+++ b/bin/mfind
@@ -104,11 +104,7 @@ function parseCmdOptions(opts, parser) {
         opts.name = new RegExp(opts.name);
     }
 
-    if (!opts._args.length) {
-        opts.paths = ['/' + opts.account + '/stor'];
-    } else {
-        opts.paths = opts._args;
-    }
+    opts.paths = opts._args;
 
     return (opts);
 }
@@ -156,6 +152,7 @@ function printEntry(opts, obj) {
         }
     });
 
+    assert(options.paths.length >= 1, 'no paths specified');
     options.paths.forEach(function (p) {
         barrier.start(p);
         client.ftw(p, options, function (err, res) {
